kubernetes/samples/scripts/domain-lifecycle/startCluster.sh

```diff
@@ -91,9 +91,12 @@ domainJson=$(${kubernetesCli} get domain ${domainUid} -n ${domainNamespace} -o j
 
 # Get server start policy for this cluster
 startPolicy=$(echo ${domainJson} | jq -r '(.spec.clusters[] | select (.clusterName == "'${clusterName}'") | .serverStartPolicy)')
+if [ "${startPolicy}" == "null" ]; then
+  startPolicy=$(echo ${domainJson} | jq -r .spec.serverStartPolicy)
+fi
```

Comment:
If the startPolicy is not set at all in either location, I think we can assume the value is IF_NEEDED.  (Please check if the other scripts also can helpfullly assume this defaulting...)

Reply:
Made changes to cluster and domain start/stop scripts to assume IF_NEEDED default policy value if not set at domain level.